psamm/command.py

```diff
@@ -91,6 +92,16 @@ def init_parser(cls, parser):
     def run(self):
         """Execute command"""
 
+    def argument_error(self, msg):
+        """Raise error indicating error parsing an argument."""
+        raise CommandError(msg)
+
+    def fail(self, exc=None):
```

Comment:
But now I provide a string to it, but it should still be OK, right? Because it should be constructed by `error_info()` function?